aopy/data.py

```diff
@@ -1069,3 +1074,34 @@ def yaml_read(filename):
         task_codes = yaml.load(file, Loader=yaml.FullLoader)
 
     return task_codes
+
+
+def get_e3v_video_frame_data( digital_data, sync_channel_idx, trigger_channel_idx, samplerate ):
+    
+    """get_e3v_video_frame_data
+
+    Compute pulse times and duty cycles from e3vision video data frames collected on an ecube digital panel.
+
+    Args:
+        digital_data (np.array): [n_time x n_channel] array of data read from ecube digital panel
```

Comment:
i prefer 
```suggestion
        digital_data (nt, nch): array of data read from ecube digital panel
```

Reply:
the format is `name (type): description`, right?

Reply:
technically it's `varIn_name1 (dim label1, dim label2): Description`
see https://analyze.readthedocs.io/en/latest/documentation-guidelines.html